itou/www/apply/views/process_views.py

```diff
@@ -474,6 +474,14 @@ def eligibility(request, job_application_id, template_name="apply/process_eligib
     if not job_application.to_siae.is_subject_to_eligibility_rules:
         raise Http404()
 
+    if request.user.is_siae_staff and (
+        suspension_explanation := job_application.to_siae.get_active_suspension_text_with_dates()
+    ):
```

Comment:
La mutation suivante ne casse aucun test :
```suggestion
    if suspension_explanation := job_application.to_siae.get_active_suspension_text_with_dates():
```

Reply:
Il faudrait un test avec un superuser pour voir une différence.
Après, je ne sais pas trop pourquoi on autorise un superuser à venir sur cette vue...

Reply:
Peut-être qu’il n’y a pas de bonnes raisons à cette vérification ?

Reply:
On va lui enlever l'accès donc je supprime la vérif :)